tools/stonith_admin.c

```diff
@@ -360,8 +360,6 @@ main(int argc, char **argv)
 
     pcmk__cli_init_logging("stonith_admin", args->verbosity);
 
-    name = strdup(crm_system_name);
```

Comment:
doesn't this finally disable the effect of '-T' option?
probably it didn't work so far being located after arg-processing though.

Reply:
Whoops, good catch, I missed that. That's actually a regression in 2.0.3.